test/unit/lib/plugins/aws/package/compile/events/cloud-watch-log.test.js

```diff
@@ -46,10 +46,18 @@ describe('AwsCompileCloudWatchLogEvents', () => {
         awsCompileCloudWatchLogEvents.serverless.service.provider.compiledCloudFormationTemplate
           .Resources.FirstLogsSubscriptionFilterCloudWatchLog1.Type
       ).to.equal('AWS::Logs::SubscriptionFilter');
+      expect(
```

Comment:
New tests should be configured via `runServerless` util, as e.g. tests in this block: https://github.com/serverless/serverless/blob/5d1c3f7ed4d481619d412b6b453d6eaf94b32b4a/test/unit/lib/plugins/aws/package/compile/events/websockets/index.test.js#L116-L258

See also: https://github.com/serverless/serverless/tree/main/test#unit-tests

In this test file let's add a new `describe` block, that will host those new tests